site/_guides/cert-manager.md

```diff
@@ -517,14 +517,13 @@ $ curl https://httpbin.davecheney.com/get
 cert-manager currently does not have a way to interact with HTTPProxy objects in order to respond to the HTTP01 challenge correctly.
 (See [#950][10] and [#951][11] for details.)
 
-cert-manager does this by creating a new, temporary Ingress object that will direct requests from Let's Encrypt to temporary pods called 'solver pods'.
+cert-manager can be configured to request certificates automatically using an `Certificate` object.
```

Comment:
nit - these first two sentences are a little confusing together. I think the point is that cert-manager can't *directly* use an HTTPProxy to generate a cert in the specified TLS secret, like it can with Ingress + annotations, but that an HTTPProxy user can directly create a `Certificate` to get one. Perhaps fine-tune the language a little bit, but it's not a big deal.

Reply:
I updated this a bit.